csharp-basic-topics/ActionAndFuncDelegates/ActionAndFuncDelegates/Messenger.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace CodeMaze

```

Comment:
Strange namespace name. Have you changed it on purpose?

Reply:
Actually, I wanted to give it a name that is somewhat related to the ownership rather than the default generated one.

Reply:
I gave it a second thought. You're right. I should not violate standard naming conventions for an arbitrary purpose. I will change that.

Reply:
Yeah, it's better to stick to the default ones for the articles. Anyways, thanks for explaining it.